plugins/modules/pnp_intent.py

```diff
@@ -875,8 +918,9 @@ class instance for further use.
                 params={"payload": reset_paramters},
                 op_modifies=True,
             )
-            self.log(str(reset_response))
+            self.log("Errored state resolution response from the API is {0}".format(str(reset_response)), "DEBUG")
```

Comment:
self.log("Response from 'update_device' API for errored state resolution: {0}".format(str(reset_response)), "DEBUG")

Reply:
Resolved